File: Solutions/IQCare.Core/IQCare/frontend/app/pmtct/_models/PatientChronicIllness.ts
export interface PatientChronicIllness {
    Id?: number;
    PatientId?: number;
    PatientMasterVisitId?: number;
    ChronicIllness?: number;
    Treatment?: string;
    Dose?: number;
    Duration?: number;
    DeleteFlag?: boolean;
    OnsetDate?: Date;
    Active?: number;
    CreateBy?: number;
}

File: Solutions/IQCare.Core/IQCare.PMTCT.Core/Models/PatientChronicIllness.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IQCare.PMTCT.Core.Models
{
    public class PatientChronicIllness
    {
        public int Id { get; set; }
        public int PatientId { get; set; }
        public int PatientMasterVisitId { get; set; }
        public int ChronicIllness { get; set; }
        public string Treatment { get; set; }
        public int Dose { get; set; }
        public int Duration { get; set; }
        public bool DeleteFlag { get; set; }
        public DateTime OnsetDate { get; set; }
        public int Active { get; set; }
        public int CreateBy { get; set; }
    }
}
